example/lib/session/foreground_session.dart

```diff
@@ -138,11 +139,18 @@ class ForegroundSession extends ChangeNotifier {
   /// It kills the background isolate, clears their authorized keys, and
   /// empties the database.
   Future<void> clear() async {
+    try {
+      var isolate = XmtpIsolate.find();
+    } catch (err) {
+      print(
+          "clear() just caught an error when trying to find an isolate: $err");
+    }
```

Comment:
I added this so we can do hot reload and be able to do a logout.
Sometimes it was trying to kill an inexistent isolate, I check for that to happen and catch the error in the other catch.